position_in_partition.hh

```diff
@@ -593,6 +593,7 @@ public:
     position_in_partition&& end() && { return std::move(_end); }
     bool contains(const schema& s, position_in_partition_view pos) const;
     bool overlaps(const schema& s, position_in_partition_view start, position_in_partition_view end) const;
+    bool is_full(const schema&) const;
```

Comment:
The name is misleading since it means a different thing from the `full()` method that returns a range including the static row.  The method you want should correspond to `all_clustered_rows()`, so maybe call it `is_all_clustered_rows()`

Reply:
Good point, fixed.

Reply:
But now I wonder if the sstable's `_position_range` should not include static rows as well... e.g. if the schema has static columns we would not set it to `all_clustered_rows()` (by default) but to `full()`, and in `set_position_range()` the left end would be `for_static_row`.

Right now the semantics of `_position_range` is unclear. It doesn't cover all possible mutation fragments from the sstable but only those that have "positions" (so clustered rows, range tombstones) *and* it special-cases the partition tombstone...